src/main/java/gold/milli/initialproject/mapper/UserServiceMapper.java

```diff
@@ -0,0 +1,13 @@
+package gold.milli.initialproject.mapper;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@RequiredArgsConstructor
+@Component
+public class UserServiceMapper {
```

Comment:
no need to this class.
just create a mapper for each service and add methods for each conversion